expo/tests/gbe/test_bid_costume.py

```diff
@@ -61,37 +88,46 @@ def test_costume_bid_post_form_not_valid(self):
         login_as(PersonaFactory().contact, self)
         data = self.get_costume_form(invalid=True)
         response = self.client.post(url)
-        nt.assert_equal(response.status_code, 200)
-        nt.assert_true('Displaying a Costume' in response.content)
+        self.assertEqual(response.status_code, 200)
+        self.assertTrue('Displaying a Costume' in response.content)
 
     def test_costume_bid_post_with_submit(self):
-        '''costume_bid, not submitting and no other problems,
+        '''costume_bid, submitting and no other problems,
         should redirect to home'''
-        url = reverse(self.view_name,
-                      urlconf="gbe.urls")
-        login_as(PersonaFactory().contact, self)
-        data = self.get_costume_form(submit=True)
-        response = self.client.post(url, data=data, follow=True)
-        nt.assert_equal(response.status_code, 200)
-        nt.assert_true(('http://testserver/gbe', 302)
-                       in response.redirect_chain)
-        nt.assert_true("Your Account" in response.content)
+        response, data = self.post_costume_submission()
+        self.assertEqual(response.status_code, 200)
```

Comment:
Well, I did say exactly the opposite of that:

> we can write nose asserts if we want, but standard python assert should work perfectly, and we should prefer them unless there's some reason to use the nose assert.

pytest understands an assertion like `assert paul in beatles` and will raise a reasonable error if it turns out that paul is not a member of that set. 

No, don't bother undoing, but moving from nose asserts to testcase asserts seems like a real loss of time for both of us.